### PR TITLE
Landing page new words

### PR DESCRIPTION
**Pull request by brnsolikyl**
Branch deployment: https://landing-page-new-words.dev.karrot.world

## What does this PR do?

Rewrites a few crucial bits of the landing page. Quick PR before new deployment

## Checklist

- [ ] added a test, or explain why one is not needed/possible...
- [ ] no unrelated changes
- [ ] asked someone for a code review
- [ ] joined [chat.foodsaving.world/channel/karrot-dev](https://chat.foodsaving.world/channel/karrot-dev)
- [ ] added an entry to CHANGELOG.md (description, pull request link, username(s))

### CODE REVIEW

**Review by nicksellen — APPROVED** [review]
Looks good! If you think it's ready, feel free to merge :)

One thing I did notice is the `karrot-screenshot.png` image has increased from 380kb to 466kb which is quite a bit bigger, so if it's possible to reduce the size that could be useful, but not essential.

**Comment by brnsolikyl** [comment]
> One thing I did notice is the `karrot-screenshot.png` image has increased from 380kb to 466kb which is quite a bit bigger, so if it's possible to reduce the size that could be useful, but not essential.

Size reduced now to 178,3 Kb

I'm going to add that About button component, then I'll merge! :)

**Comment by brnsolikyl** [comment]
I feel it's ready to merge now after adding the About Karrot component. But maybe a quick check before merging would be good, @nicksellen , just to make sure I didn't do anything ugly

**Comment by brnsolikyl** [comment]
Not sure what happened there. I ran `yarn test -u storyshots`, it got updated but still not passing the test...

File: src/base/components/KAbout.vue
Comment (nicksellen) [inline src/base/components/KAbout.vue:25]: Text should be put into translation system.

File: src/base/pages/Landing.vue
Comment (nicksellen) [inline src/base/pages/Landing.vue:119]: The button doesn't look quite fitting in style to me. The simplest change I think it just to add `unelevated` attribute. Still looks a bit different from the other buttons, but I think it's OK.

aboutbutton

File: src/locales/locale-en.json
Comment (nicksellen) [inline src/locales/locale-en.json:21]: As this line no longer uses the `{code}` variable, it can be removed from the usage too.

So this:

https://github.com/yunity/karrot-frontend/blob/18cff7b940f66bddb3f80e0f5baf40e336b87dcc/src/base/pages/Landing.vue#L106-L114

and be changed to:

```html
<p v-t="'ABOUT_KARROT.SECTIONS.DEMOCRATIC.DESCRIPTION2'" />
```

**Review by nicksellen — CHANGES_REQUESTED** [review]
Looks good! A couple of small things left though...

... regarding the tests, it fails for me locally as it is now, and `yarn test -u storyshots` fixes it here. Maybe you forgot to check in the changed files?

File: src/base/components/KAbout.vue
Comment (nicksellen) [reply src/base/components/KAbout.vue:25]: @brnsolikyl mentioned that it maybe makes sense to keep it in English, given the "team" is only communicating in English. Makes sense to me, but would also include a code-level comment to explain why it's not translated, e.g.:

```html
<!-- this is in English because -->
```

File: src/base/pages/Landing.vue
Comment (brnsolikyl) [reply src/base/pages/Landing.vue:119]: I added unelevated. For some reason I like it being different :) But I don't mind changing to fit the style
This button will be temporary anyhow if we go on to create the about page and improving the landing page

**Comment by brnsolikyl** [comment]
> .. regarding the tests, it fails for me locally as it is now, and `yarn test -u storyshots` fixes it here. Maybe you forgot to check in the changed files?

How would I do that? I ran it and committed the first time. Now, whenever I run it, it passes the test and no files are changed.

**Comment by nicksellen** [comment]
I think your dependencies are not up to date locally, and the newer versions (e.g. quasar) generate slightly different html, hence the change in the snapshots across many different components unrelated to the change.

To updated dependencies just run `yarn`... then re-run tests / snapshot updating. I think that might be it :crossed_fingers:

**Review by nicksellen — APPROVED** [review]
Great, all looks good to me now! :+1:

**Pull request merged**
